### PR TITLE
UIDATIMP-801: Toggles should display currency selected in Currency select in Extended information accordion

### PR DESCRIPTION
**Pull request by IhorKoba**
## Purpose

Add dependency for all toggles in Field mapping Invoice Details accordions on Currency select in
Extended information accordion

## Approach

- Get value of currency field from redux-form state and pass it to related accordions
- Apply currency prop to `TypeToggle`  components

## Refs

https://issues.folio.org/browse/UIDATIMP-801

## Screenshots

image

### CODE REVIEW

File: src/settings/MappingProfiles/detailsSections/edit/MappingInvoiceDetails.js [hunk 1]
```diff
@@ -17,18 +17,25 @@ import {
...
+  mappingDetails,
   initialFields,
   referenceTables,
   setReferenceTables,
   getRepeatableFieldAction,
   okapi,
 }) => {
+  const getCurrencyFromDetails = state => state?.mappingFields
+    ?.find(item => (item.name === CURRENCY_FIELD))?.value
+    .replace(/['"]+/g, '');
```
Comment (mariia-aloshyna) [inline src/settings/MappingProfiles/detailsSections/edit/MappingInvoiceDetails.js:33]:
```suggestion
  const currencyFromDetails = mappingDetails
    ?.mappingFields
    ?.find(item => (item.name === CURRENCY_FIELD))?.value
    .replace(/['"]+/g, '');
```

File: src/settings/MappingProfiles/detailsSections/edit/MappingInvoiceDetails.js [hunk 2]
```diff
@@ -17,18 +17,25 @@ import {
...
+
   const adjustments = referenceTables?.adjustments || [];
   const fundDistributions = referenceTables?.fundDistributions || [];
   const lineAdjustments = referenceTables?.lineAdjustments || [];
+  const currency = getCurrencyFromDetails(mappingDetails);
```
Comment (mariia-aloshyna) [inline src/settings/MappingProfiles/detailsSections/edit/MappingInvoiceDetails.js:38]:
```suggestion
```

File: src/settings/MappingProfiles/detailsSections/edit/MappingInvoiceDetails.js [hunk 3]
```diff
@@ -38,6 +45,7 @@ export const MappingInvoiceDetails = ({
       />
       <InvoiceAdjustments
         adjustments={adjustments}
+        currency={currency}
```
Comment (mariia-aloshyna) [inline src/settings/MappingProfiles/detailsSections/edit/MappingInvoiceDetails.js:48]:
```suggestion
        currency={currencyFromDetails}
```

File: src/settings/MappingProfiles/detailsSections/edit/MappingInvoiceDetails.js (hunk 1 above)
Comment (IhorKoba) [reply src/settings/MappingProfiles/detailsSections/edit/MappingInvoiceDetails.js]: done

File: src/settings/MappingProfiles/detailsSections/edit/MappingInvoiceDetails.js (hunk 2 above)
Comment (IhorKoba) [reply src/settings/MappingProfiles/detailsSections/edit/MappingInvoiceDetails.js]: done

File: src/settings/MappingProfiles/detailsSections/edit/MappingInvoiceDetails.js (hunk 3 above)
Comment (IhorKoba) [reply src/settings/MappingProfiles/detailsSections/edit/MappingInvoiceDetails.js]: done

**Review by mariia-aloshyna — APPROVED** [review]

**Pull request merged**